### PR TITLE
Create terrama2 user

### PR DESCRIPTION
**Pull request by janosimas**

### CODE REVIEW

File: scripts/postinst.in [hunk 1]
```diff
@@ -68,6 +68,14 @@ if [ $? != 0 ]; then
   valid $? "Error adding pm2 to startup."
 fi
 
+# Create a terrama2 user
+# and default data folder
+(
+  adduser --system terrama2
```
Comment (raphaelrpl) [inline scripts/postinst.in]: This command seems to create user home. Is it necessary? Maybe giving option `--no-create-home`.

Comment (janosimas) [inline scripts/postinst.in]: I'll try that :+1:

**Comment by janosimas** [comment]
jenkins rebuild

**Comment by janosimas** [comment]
jenkins rebuild

**Pull request merged**